docs/content/v2/en/options.md

```diff
@@ -189,6 +189,31 @@ List of all available font files of a font family variation.
 | `src`  | `String` | yes      | path to a font file, the use of aliases is possible        |
 | `type` | `String` | yes      | file format of the specified file, e.g. `woff`, `woff2`, … |
 
+## `pictureFormats`
+- Type: `Array`
+  - Default: `['webp', 'avif', 'jpg|jpeg|png|gif']`
+
+Defines the default `formats` for the `SpeedkitPicture`.
```

Comment:
The attribute "pictureFormats" defines the image formats to be generated for SpeedkitPicture. The order defines the loading priority of the formats in the browser.

Reply:
The format order is defined separately in `SpeedkitPicture` 😉

https://github.com/GrabarzUndPartner/nuxt-speedkit/blob/a13d828aa0608a8b0d6e8d675b965d60b39e7ec0/lib/components/SpeedkitPicture/Base.vue#L22